### ISSUE TITLE
Maximize causes rendering distortion

### ISSUE THREAD

**Issue by alecov** [laurent22/joplin#9657]
### Operating system

Linux

### Joplin version

2.13.12

### Desktop version info

Joplin 2.13.12 (prod, linux)

Client ID: 1a99c0bf6a7a4a2baeddad857f710766
Sync Version: 3
Profile Version: 44
Keychain Supported: No

Revision: e027cdc

### Current behaviour

Running Joplin Desktop via AppImage on Linux (MATE desktop environment), this happens:

1. Maximize Joplin
2. Close Joplin
3. Reopen
4. Attached screenshot happens

Basically the rendering becomes warped and shifted (menus and stuff must be clicked at where they should be in the window, they are rendered at an offset). Also, the window fixes to the foreground and covers everything, Joplin needs to be killed from tty.
![Screenshot at 2024-01-04 11-53-58](https://github.com/laurent22/joplin/assets/20651620/2c26b126-adf5-4086-a876-01fd3b685cba)

**Comment by alecov** [comment]
Not fixed.

**Comment by alecov** [comment]
Not fixed.
